docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/node--cgov-home-landing--ncid-with-title--full.html.twig

```diff
@@ -0,0 +1,12 @@
+{# Title Row #}
+<div class="text-center padding-y-6">
+    {% set hr_attribs = create_attribute({}) %}
+    {% if is_front == TRUE %}
+      {% set hr_attribs = hr_attribs.addClass('show-for-sr') %}
```

Comment:
This conditional should not be needed. The front page will just use the template without a title.